tests/virtualization/universal/register_guests.pm

```diff
@@ -16,13 +16,20 @@ use version_utils;
 sub run_test {
     my ($self) = @_;
     # Use serial terminal, unless defined otherwise. The unless will go away once we are certain this is stable
-    $self->select_serial_terminal unless get_var('_VIRT_SERIAL_TERMINAL', 1) == 0;
+    # $self->select_serial_terminal unless get_var('_VIRT_SERIAL_TERMINAL', 1) == 0;
+    select_console 'root-console';
+    my %ltss_products = @{get_var_array("LTSS_PRODUCTS")};
```

Comment:
the setting name, LTSS_PRODUCTS, is a little confusing. It looks like the product list instead of scc codes.

Reply:
@tonyyuan1  Agree, better to rename the variable. Besides, where do you put the setting? In testsuite qam-kvm-install and qam-xen-install's settings, rather than job group or medium settings, right?

Reply:
It includes info about both product and scc code .  The key represents product and its value is ltss scc code. A key is used to check if a guest os is ltss product. If yes, register with the corresponding code. 

Is LTSS_PRODUCTS reasonable? or any better one?

Yes,  In testsuite qam-kvm-install and qam-xen-install's settings. It is only used by the two.